spec/javascripts/support/jasmine.yml

```diff
@@ -0,0 +1,30 @@
+# path to parent directory of src_files
+# relative path from Rails.root
+# defaults to app/assets/javascripts
+src_dir: "app/assets/javascripts"
+
+# list of file expressions to include as source files
+# relative path from src_dir
+src_files:
+  - "../../../spec/javascripts/helpers/jquery-3.5.1.js"
```

Comment:
Where does this come from? I don't think it's needed and it's a very different version of jQuery to what GOV.UK uses 1.12